content/blog/2019/07/2019-07-15-pipeline-config-history-plugin.adoc

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: "Introducing the Pipeline Configuration History Plugin"
+tags:
+- pipeline
+- plugins
+author: pch-maintainers
+---
+
+https://jenkins.io/doc/book/pipeline/[Pipelines] are the efficient and modern way how to create jobs in Jenkins.
+To recognize changes of pipelines in a fast and easy way, we developed the plugin plugin:pipeline-config-history[Pipeline Configuration History].
```

Comment:
Minor phrasing change, feel free to ignore it if it is uncomfortable or too invasive.
```suggestion
To recognize pipeline changes quickly and easily, we developed the plugin:pipeline-config-history[Pipeline Configuration History] plugin.
```